doc/contribute/committer_expectations.rst

```diff
@@ -0,0 +1,165 @@
+.. _committer-expectations:
+
+Committer Expectations
+######################
+
+Overview
+********
+
+The Zephyr project encourages you to submit changes as small pull-requests.
+Small pull requests (PRs) have the following benefits:
+
+- Reviewed more quickly and reviewed more thoroughly. It's easier for reviewers
+  to set aside a few minutes to review small changes several times than it is to
+  allocate a 30 minute block to review a large PR.
```

Comment:
It seems minor, but I think it is worth discussing whether this number should remain here.I don't want people to get the mistaken impression that 30 minutes is the worst case scenario for a reviewer. 

In my experience, some reviews take multiple hours or even days if you're going to do them carefully and not just rubber-stamp.

I would suggest something like "allocate a large block of time" for parallelism with the "review a large PR" language.